src/chocolatey.resources/helpers/functions/Get-WmiObject.ps1

```diff
@@ -0,0 +1,15 @@
+﻿## Get-WmiObject
+##############################################################################################################
+## Wrapper function. Replaces "Win32_Product" class with Get-Installer helper.
+##############################################################################################################
+
+function Get-WmiObject {
```

Comment:
Does this work without specifying any params? That would be a definite improvement on what I was doing before.